tests/requests.rs

```diff
@@ -68,6 +68,35 @@ impl OsuSingleton {
 
         Ok(self.inner.wait().lock().await)
     }
+
+    async fn get_oauth(&self) -> Result<MutexGuard<'_, Osu>> {
+        let cmp_res = self
+            .initialized
+            .compare_exchange(false, true, SeqCst, SeqCst);
+
+        if cmp_res.is_ok() {
+            tracing_subscriber::fmt()
+                .with_writer(TestWriter::new())
+                .with_env_filter(EnvFilter::builder().parse("rosu_v2=trace,info").unwrap())
+                .init();
+            dotenv().ok();
+
+            let access_token = env::var("OSU_ACCESS_TOKEN")
+                .wrap_err("missing OSU_ACCESS_TOKEN")?;
```

Comment:
Access tokens are only temporarily valid so this would require regular manual updates to pass the test.